Makefile

```diff
@@ -18,7 +18,11 @@ TIMEOUT ?= 14400s
 # TEST_EXEC_NODES=1, otherwise by default the specs are run in parallel on 4 ginkgo test node.
```

Comment:
@rnapoles-rh Could you please add IBM cluster test node info in the comment as well.